docs/website/versioned_docs/version-3.0.0/command-reference/logs.md

```diff
@@ -10,159 +10,269 @@ directory.
 ## Running the command 
 
 If you haven't already done so, you must [initialize](../command-reference/init) your source code with the `odo 
-init` command. Next, run the `odo dev` command so that odo can create the resources on the Kubernetes cluster.
```

Comment:
I find this reference very verbose for a reference, and not going straight to the point. It would be more appropriate as a base for a blog post or part of the "getting started" section.

Reply:
GH shows you are pointing to a deleted line. Can you point out which line in the docs you are referring to? My guess is that you are referring to:
> If you haven't already done so, you must [initialize](https://deploy-preview-5825--odo-docusaurus-preview.netlify.app/docs/3.0.0/command-reference/init) your source code with the odo init command. This document uses [this git project](https://github.com/odo-devfiles/nodejs-ex) and the following devfile to explain the flow of things in odo logs command:

Reply:
My intention was to point to the whole page

Reply:
I have trimmed it. PTAL.